tools/pfe-tools/config.ts

```diff
@@ -36,7 +36,7 @@ export interface PfeConfig {
 
 const SITE_DEFAULTS: Required<SiteOptions> = {
   description: 'PatternFly Elements: A set of community-created web components based on PatternFly design.',
-  favicon: '/docs/images/logo/pfe-icon-blue.svg',
+  favicon: '/docs/images/favicon.ico',
```

Comment:
instead of the .ico, can we use an svg here?

Reply:
@bennypowers do you mean using `pfe-logo-icon-white.svg` ? It is a pure white variant of the logo.

Reply:
WDYT about this one? https://github.com/patternfly/patternfly-elements/blob/main/brand/logo/svg/pf-logo-small.svg